src/styles/partials/components/_modal.scss

```diff
@@ -0,0 +1,151 @@
+.dg_modal {
+  .dg_icon-link {
+    padding: 0;
+    margin: 0;
+    border: 0;
+  }
+
+  .dg_modal__inner {
+    position: relative;
+    padding: $padding-largest;
+    margin-top: $margin-small;
+    margin-bottom: $margin-small;
+    width: auto;
+    height: auto;
+    opacity: 1;
+    color: $white;
+
+    p {
+      letter-spacing: 0;
+      color: $white;
+      opacity: 1;
+      text-align: center;
+      font: italic 28px/44px $default-body-font;
+      margin-bottom: 60px;
+    }
+  }
+}
+@keyframes fade-in {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+@keyframes appear {
+  from {
+    transform: translate(-50%, -40%);
+    opacity: 0;
+  }
+  to {
+    transform: translate(-50%, -50%);
+    opacity: 1;
+  }
+}
+[data-a11y-dialog-native] {
+  .dialog-overlay {
+    display: none;
+  }
+}
+dialog[open] {
+  display: block;
+  background: $pale-blue;
+}
+.dialog[aria-hidden="true"] {
+  display: none;
+}
+.dialog-overlay {
+  z-index: 2;
```

Comment:
@sgrg1 I Looked in to the item we discussed at your desk where the overlay would phase in when the prototype opened in a new window. Setting the `z-index` of this line to -1 resolved the issue. This could also potentially be resolved with adjusting the display property, but I haven't had the chance to test this option.

Reply:
change to  z-index =-1